planner/core/testdata/integration_serial_suite_in.json

```diff
@@ -38,6 +38,27 @@
       "explain format = 'brief' select count(*) from fact_t where d1_k not in (select d1_k from d1_t)"
     ]
   },
+  {
+    "name": "TestMPPOuterJoinBuildSideForBroadcastJoin",
+    "cases": [
```

Comment:
add more case for other conditions, cross join?

also need randgen tests to guarantee rightness.

Reply:
If the outer join has other condition, it will always use inner table as the build side, this behavior is not changed by this pr. For cross join, TiFlash will convert `Cross right join` to `Cross left join`, so is not affected by this pr.